virttest/vt_utils/block.py

```diff
@@ -0,0 +1,198 @@
+#
+# library for the block related helper functions
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; specifically version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: Red Hat (c) 2023 and Avocado contributors
+# Author: Houqi Zuo <hzuo@redhat.com>
+import platform
+import re
+
+from avocado.utils import process
+from avocado.utils import wait
+from virttest import utils_numeric
+from virttest.vt_utils import filesystem
+
+PARTITION_TYPE_PRIMARY = "primary"
+
+
+def get_disks(partition=False):
+    """
+    List all disks or disks with no partition.
+
+    :param partition: if true, list all disks; otherwise,
+                      list only disks with no partition
+    :type partition: boolean
+    :return: the disks info.( e.g. {kname: [kname, size, type, serial, wwn]} )
+    :rtype: dict
+    """
+    disks_dict = {}
+    parent_disks = set()
+    driver = "pci"
+    if platform.machine() == "s390x":
+        driver = "css0"
+    block_info = process.run('ls /sys/dev/block -l | grep "/%s"'
+                             % driver).stdout_text
```

Comment:
```suggestion
    block_info = process.run('ls /sys/dev/block -l | grep "/%s"'
                             % driver, verbose=False).stdout_text
```
Suggest that we do not log any information for the utility module.
Refer to https://github.com/avocado-framework/avocado/blob/cfce41795143b3e32f77e3480680a0346149b171/avocado/utils/process.py#L949

Reply:
> , verbose=False

Done. Please check again, thanks!